index.js

```diff
@@ -1,12 +1,11 @@
 const fs = require('fs');
-const numApproved = 2;
 
 module.exports = robot => {
 	robot.on('pull_request.opened', async context => {
 		const githubConfig = context.repo({path: 'Github.json'})
 	    const res = await context.github.repos.getContent(githubConfig)
 	    const template = JSON.parse(Buffer.from(res.data.content, 'base64').toString());
-	    console.log(JSON.stringify(template));
+	    var numApproved = template["approved-reviewer-count"];
```

Comment:
You should use [parseInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) here to make sure that the value for that field is a number